apps/vr-tests/.storybook/preview.js

```diff
@@ -2,6 +2,8 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28777&commitId=ada9594acdc6fadb556c0ed32c86d0c4704a9323&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|0| Added|
|react-charting-AreaChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-AreaChart.Basic.default.chromium.png|0| Added|
|react-charting-AreaChart.Basic.hover.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility - Dark Mode.default.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility - RTL.default.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|0| Added|
|react-charting-AreaChart.Multiple - Dark Mode.default.chromium.png|0| Added|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|0| Added|
|react-charting-AreaChart.Multiple - RTL.default.chromium.png|0| Added|
|react-charting-AreaChart.Multiple.default.chromium.png|0| Added|
|react-charting-AreaChart.Multiple.hover.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-DonutChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-DonutChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-DonutChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-DonutChart.Basic.default.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic - Dark Mode.default.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic - RTL.default.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-GaugeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-GaugeChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-GaugeChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-GaugeChart.Basic.default.chromium.png|0| Added|
|react-charting-GaugeChart.Variant - Dark Mode.default.chromium.png|0| Added|
|react-charting-GaugeChart.Variant - RTL.default.chromium.png|0| Added|
|react-charting-GaugeChart.Variant.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-HeatMapChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-HeatMapChart.Basic.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-HorizontalBarChart</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic.hover.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant - Dark Mode.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant - RTL.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Axis - Dark Mode.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Axis - RTL.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Axis.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Benchmark - Dark Mode.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Benchmark - RTL.default.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Benchmark.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-LineChart</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|0| Added|
|react-charting-LineChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-LineChart.Basic.default.chromium.png|0| Added|
|react-charting-LineChart.Basic.hover.chromium.png|0| Added|
|react-charting-LineChart.Events - Dark Mode.default.chromium.png|0| Added|
|react-charting-LineChart.Events - RTL.default.chromium.png|0| Added|
|react-charting-LineChart.Events.default.chromium.png|0| Added|
|react-charting-LineChart.Gaps - Dark Mode.default.chromium.png|0| Added|
|react-charting-LineChart.Gaps - RTL.default.chromium.png|0| Added|
|react-charting-LineChart.Gaps.default.chromium.png|0| Added|
|react-charting-LineChart.Multiple - Dark Mode.default.chromium.png|0| Added|
|react-charting-LineChart.Multiple - RTL.default.chromium.png|0| Added|
|react-charting-LineChart.Multiple.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-MultiStackBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-MultiStackBarChart.Basic_Absolute - Dark Mode.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_Absolute - RTL.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_Absolute.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_PartToWhole - Dark Mode.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_PartToWhole - RTL.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_PartToWhole.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With_Placeholders - Dark Mode.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With_Placeholders - RTL.default.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With_Placeholders.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-PieChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-PieChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-PieChart.Basic.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-SankeyChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-SankeyChart.Basic.default.chromium.png|0| Added|
|react-charting-SankeyChart.PlaceHolder - Dark Mode.default.chromium.png|0| Added|
|react-charting-SankeyChart.PlaceHolder - RTL.default.chromium.png|0| Added|
|react-charting-SankeyChart.PlaceHolder.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-SparkLineChart.Basic.default.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Dark Mode.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Basic - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Basic.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Dynamic - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Dynamic.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label - Dark Mode.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - Dark Mode.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - RTL.default.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>